gulpfile.js

```diff
@@ -52,12 +52,18 @@ function js() {
     .pipe(dest('./html/js/dist', { sourcemaps: true }))
 }
 
+function buildJs() {
+  return src([
+    './html/js/display*.js',
+    './html/js/product-multilingual.js',
+    './html/js/search.js'
+  ], { sourcemaps: true })
+  .pipe(dest('./html/js/dist', { sourcemaps: true }))
+}
+
 function jQueryUiThemes() {
   return src('./node_modules/jquery-ui-dist/**/*.css', { sourcemaps: true })
     .pipe(dest('./html/css/dist/jqueryui/themes/base', { sourcemaps: true }))
 }
 
-exports.js = js
```

Comment:
Why did you remove exports? Without them we cannot run tasks one by one.

Reply:
True - I actually never did that, so I supposed they weren't required. I'll put them back in. :)